main.go

```diff
@@ -1033,6 +1033,7 @@ func getBuildParameters(conf *config, build *buildOptions) ([]*gitlab.PipelineVa
 	// is master, in which case we rely on the default.
 	if build.repo == "meta-mender" && build.baseBranch != "master" {
 		buildParameters = append(buildParameters, &gitlab.PipelineVariable{Key: repoToBuildParameter("poky"), Value: build.baseBranch})
+		buildParameters = append(buildParameters, &gitlab.PipelineVariable{Key: repoToBuildParameter("meta_openembedded"), Value: build.baseBranch})
```

Comment:
Almost, it should be:
```suggestion
		buildParameters = append(buildParameters, &gitlab.PipelineVariable{Key: repoToBuildParameter("meta-openembedded"), Value: build.baseBranch})
```

And once we are on it, please add also `meta-raspberrypi`; it won't have an effect as per today, but for consistency and just in case we start building Rpi in the future by default.